Installation issue: Faodel 1.1906.1 GCC 9.3.0 Ubuntu 20.04

Using `spack@develop` with commit id `70c3b0ba0941` from 3-June-2020.

System:
* Ubuntu 20.04
* GCC 9.3.0
* x86_64 and ppc64le

```
$> spack install faodel
...
3 errors found in build log:
     2302    [ 90%] Built target mpi_dirman_op_messages
     2303    [ 90%] Building CXX object src/kelpie/CMakeFiles/kelpie.dir/pools/RFTPool/RFTPool.cpp.o
     2304    cd /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/kelpie && /opt/spack/lib/spack/env/gcc/g++
               -DBOOST_LOG_DYN_LINK -Dkelpie_EXPORTS -I/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/src -I/tm
             p/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src -I/tmp/root/spack-stage/spack-stage-faodel-1.190
             6.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/tpl/cereal/include -I/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2w
             vr37z2/spack-src/tpl/gperftools/src -I/super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost
             -1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/include -I/super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gc
             c-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/include  -O2 -g -fPIC   -Wno-unused-parameter -Wno-unused-local-typedefs -std=c++11 -o CMak
             eFiles/kelpie.dir/pools/RFTPool/RFTPool.cpp.o -c /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/sr
             c/kelpie/pools/RFTPool/RFTPool.cpp
     2305    [ 91%] Building CXX object src/kelpie/CMakeFiles/kelpie.dir/services/PoolServerDriver.cpp.o
     2306    cd /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/kelpie && /opt/spack/lib/spack/env/gcc/g++
               -DBOOST_LOG_DYN_LINK -Dkelpie_EXPORTS -I/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/src -I/tm
             p/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src -I/tmp/root/spack-stage/spack-stage-faodel-1.190
             6.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/tpl/cereal/include -I/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2w
             vr37z2/spack-src/tpl/gperftools/src -I/super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost
             -1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/include -I/super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gc
             c-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/include  -O2 -g -fPIC   -Wno-unused-parameter -Wno-unused-local-typedefs -std=c++11 -o CMak
             eFiles/kelpie.dir/services/PoolServerDriver.cpp.o -c /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-sr
             c/src/kelpie/services/PoolServerDriver.cpp
     2307    /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/src/kelpie/pools/DHTPool/DHTPool.cpp: In lambda fun
             ction:
  >> 2308    /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-src/src/kelpie/pools/DHTPool/DHTPool.cpp:232:75: error:
              lambda parameter 'key' previously declared as a capture
     2309      232 |                  [&key, &returned_ldo, &cv, &is_found] (bool success, Key key, lunasa::DataObject result_ldo,
     2310          |                                                                       ~~~~^~~
  >> 2311    make[2]: *** [src/kelpie/CMakeFiles/kelpie.dir/build.make:385: src/kelpie/CMakeFiles/kelpie.dir/pools/DHTPool/DHTPool.cpp.o] Error 1
     2312    make[2]: *** Waiting for unfinished jobs....
     2313    [ 91%] Linking CXX executable tb_dirman_corecentralized
     2314    cd /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/tests/dirman && /super/absurdly/ridiculously/o
             bnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.17.1-zzq5x24vyoljvoranxaos6j5plt2vu5e/bin/cmake -E cmake_link_scr
             ipt CMakeFiles/tb_dirman_corecentralized.dir/link.txt --verbose=1
     2315    /opt/spack/lib/spack/env/gcc/g++  -O2 -g   CMakeFiles/tb_dirman_corecentralized.dir/unit/tb_dirman_corecentralized.cpp.o  -o tb_dirman_corecentr
             alized  -Wl,-rpath,/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/dirman:/super/absurdly/rid
             iculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/googletest-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib:/super/
             absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib:
             /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/opbox:/tmp/root/spack-stage/spack-stage-faode
             l-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/lunasa:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wv
             r37z2/spack-build/tpl/gperftools:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/nnti:/tmp/ro
             ot/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/faodel-services:/super/absurdly/ridiculously/obnoxio
             usly/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/lib:/tmp/root/spack-stage/spack-st
             age-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/whookie:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4j
             e2dr33l2wvr37z2/spack-build/src/faodel-common:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src
             /sbl ../../src/dirman/libdirman.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/googlete
             st-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib/libgtest.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x
             86_64/gcc-9.3.0/googletest-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib/libgtest_main.so /super/absurdly/ridiculously/obnoxiously/long/path/for/r
             elocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_program_options-mt.so /super/absurdly/ri
             diculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_se
             rialization-mt.so -lz -ldl ../../src/opbox/libopbox.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_6
             4/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_serialization-mt.so ../../src/lunasa/liblunasa.so ../../tpl/gperftools/li
             btcmalloc.so ../../tpl/gperftools/libspinlock.so ../../src/nnti/libnnti.so ../../src/faodel-services/libfaodel-services.so /super/absurdly/ridic
             ulously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/lib/libmpicxx.so /s
             uper/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/
             lib/libmpi.so ../../src/whookie/libwhookie.so ../../src/faodel-common/libfaodel-common.so ../../src/sbl/libsbl.so /super/absurdly/ridiculously/o
             bnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_log-mt.so /sup
             er/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/l
             ib/libboost_log_setup-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cn
             tjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_filesystem-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04
             -x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_thread-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for
             /relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_atomic-mt.so /super/absurdly/ridiculou
             sly/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_date_time
             -mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxo
             d7mbhfkzm/lib/libboost_regex-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.
             73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_chrono-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20
             .04-x86_64/gcc-9.3.0/googletest-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib/libgtest.so -lpthread
     2316    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build'
     2317    [ 91%] Built target tb_dirman_corecentralized
     2318    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build'
  >> 2319    make[1]: *** [CMakeFiles/Makefile2:1162: src/kelpie/CMakeFiles/kelpie.dir/all] Error 2
     2320    make[1]: *** Waiting for unfinished jobs....
     2321    [ 92%] Linking CXX executable tb_dirman_directorycache
     2322    cd /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/tests/dirman && /super/absurdly/ridiculously/o
             bnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.17.1-zzq5x24vyoljvoranxaos6j5plt2vu5e/bin/cmake -E cmake_link_scr
             ipt CMakeFiles/tb_dirman_directorycache.dir/link.txt --verbose=1
     2323    /opt/spack/lib/spack/env/gcc/g++  -O2 -g   CMakeFiles/tb_dirman_directorycache.dir/unit/tb_dirman_directorycache.cpp.o  -o tb_dirman_directoryca
             che  -Wl,-rpath,/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/dirman:/super/absurdly/ridicu
             lously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/googletest-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib:/super/abs
             urdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib:/tm
             p/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/opbox:/tmp/root/spack-stage/spack-stage-faodel-1
             .1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/lunasa:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37
             z2/spack-build/tpl/gperftools:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/nnti:/tmp/root/
             spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/faodel-services:/super/absurdly/ridiculously/obnoxiousl
             y/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/lib:/tmp/root/spack-stage/spack-stage
             -faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/whookie:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2d
             r33l2wvr37z2/spack-build/src/faodel-common:/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build/src/sb
             l ../../src/dirman/libdirman.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/googletest-
             1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib/libgtest.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_
             64/gcc-9.3.0/googletest-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib/libgtest_main.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relo
             cation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_program_options-mt.so /super/absurdly/ridic
             ulously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_seria
             lization-mt.so -lz -ldl ../../src/opbox/libopbox.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/g
             cc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_serialization-mt.so ../../src/lunasa/liblunasa.so ../../tpl/gperftools/libtc
             malloc.so ../../tpl/gperftools/libspinlock.so ../../src/nnti/libnnti.so ../../src/faodel-services/libfaodel-services.so /super/absurdly/ridiculo
             usly/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/lib/libmpicxx.so /supe
             r/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.2.1-5cek5pmy4xtwzhk3qf52gsubgt3zsubx/lib
             /libmpi.so ../../src/whookie/libwhookie.so ../../src/faodel-common/libfaodel-common.so ../../src/sbl/libsbl.so /super/absurdly/ridiculously/obno
             xiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_log-mt.so /super/
             absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/
             libboost_log_setup-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjv
             ey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_filesystem-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x8
             6_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_thread-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/re
             location/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_atomic-mt.so /super/absurdly/ridiculously
             /obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_date_time-mt
             .so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.0-cntjvey3hgfap5gomrknxxod7m
             bhfkzm/lib/libboost_regex-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04-x86_64/gcc-9.3.0/boost-1.73.
             0-cntjvey3hgfap5gomrknxxod7mbhfkzm/lib/libboost_chrono-mt.so /super/absurdly/ridiculously/obnoxiously/long/path/for/relocation/linux-ubuntu20.04
             -x86_64/gcc-9.3.0/googletest-1.10.0-kvnfodbmoslfazk737obfrmhtrscsl6q/lib/libgtest.so -lpthread
     2324    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build'
     2325    [ 92%] Built target tb_dirman_directorycache

See build log for details:
  /tmp/root/spack-stage/spack-stage-faodel-1.1906.1-c7kkkhtjtudd4sl4je2dr33l2wvr37z2/spack-build-out.txt
```

Full build log: [faodel.txt](https://github.com/spack/spack/files/4724653/faodel.txt)
Configure arguments: [faodel-configure.txt](https://github.com/spack/spack/files/4724664/faodel-configure.txt)

@tkordenbrock

--- a/var/spack/repos/builtin/packages/faodel/lambda-capture-f0267fc.patch
+++ b/var/spack/repos/builtin/packages/faodel/lambda-capture-f0267fc.patch
@@ -1,0 +1,26 @@
+From f0267fc728d0f49ad396b83e8e62fba54027f31f Mon Sep 17 00:00:00 2001
+From: Craig Ulmer <craig@craigulmer.com>
+Date: Fri, 29 May 2020 23:08:29 -0700
+Subject: [PATCH] FIX: Removes variable from lambda capture that conflicted
+ with args
+
+---
+ src/kelpie/pools/DHTPool/DHTPool.cpp | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/kelpie/pools/DHTPool/DHTPool.cpp b/src/kelpie/pools/DHTPool/DHTPool.cpp
+index df9c1d3..d24aa89 100644
+--- a/src/kelpie/pools/DHTPool/DHTPool.cpp
++++ b/src/kelpie/pools/DHTPool/DHTPool.cpp
+@@ -229,7 +229,7 @@ rc_t DHTPool::Need(const Key &key, size_t expected_ldo_user_bytes, lunasa::DataO
+   bool is_found=false;
+ 
+   rc_t rc = Want(key, expected_ldo_user_bytes,
+-                 [&key, &returned_ldo, &cv, &is_found] (bool success, Key key, lunasa::DataObject result_ldo,
++                 [&returned_ldo, &cv, &is_found] (bool success, Key key, lunasa::DataObject result_ldo,
+                                                         const kv_row_info_t &ri, const kv_col_info_t &c) {
+       if(success) {
+         *returned_ldo = result_ldo;
+-- 
+2.24.2 (Apple Git-127)
+

--- a/var/spack/repos/builtin/packages/faodel/package.py
+++ b/var/spack/repos/builtin/packages/faodel/package.py
@@ -55,6 +55,7 @@ class Faodel(CMakePackage):
     patch('faodel_mpi.patch', when='@1.1811.1 ~mpi')
     # FAODEL Github issue #5
     patch('faodel_sbl.patch', when='@1.1811.1 logging=sbl')
+    patch('lambda-capture-f0267fc.patch', when='@1.1906.1')
 
     def cmake_args(self):
         spec = self.spec

